lib/pacemaker/pcmk_output.c

```diff
@@ -1884,10 +1884,30 @@ rule_check_default(pcmk__output_t *out, va_list args)
             return out->info(out, "Rule %s does not satisfy conditions",
                              rule_id);
         default:
-            return out->info(out,
-                             "Could not determine whether rule %s is expired",
-                             rule_id);
+            break;
+    }
+
+    /* text_err() prints to stderr, hard-coded. We can't call out->err() here
+     * because we don't want a newline.
+     */
+    fprintf(stderr, "Could not determine whether rule %s is expired: ",
+            rule_id);
```

Comment:
For future-proofing (vs new or reimplemented formats) I'd rather just duplicate this in the out->err() calls. That also lets us merge the switches and simplify the default

Reply:
Good point